### PR TITLE
ps: cleanup, add tests

### PR DESCRIPTION
**Pull request by rminnich**
note: I realize this is not  up to our current standards, but it now at least passes
golint
go vet
and it sort of has a test. 

We'd never let it go in this way today but ... well ... be gentle.
I think if it's even an improvement we should start here and I'll keep at it.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
fork bomb? no idea why

File: cmds/ps/ps_test.go [hunk 1]
```diff
@@ -7,18 +7,171 @@
...
+		}
+		for n, f := range tt.files {
+			procf := filepath.Join(pd, n)
+			t.Logf("Write %v", procf)
+			if err := ioutil.WriteFile(procf, []byte(f), 0666); err != nil {
+				t.Fatal(err)
+			}
+		}
+		c := testutil.Command(t, "aux")
+		c.Env = []string{"UROOT_PSPATH=" + d}
```
Comment (hugelgupf) [inline cmds/ps/ps_test.go]: c.Env = append(c.Env, ...)

That'll solve the fork bomb.

**Comment by rminnich** [comment]
Fork bomb solved, thanks

**Comment by rminnich** [comment]
The tests all seem to work too.

**Comment by rminnich** [comment]
This is not done but much improved, I'd like to get this in as a checkpoint change before I continue.

**Pull request merged**